selftests/deployment/README.rst

```diff
@@ -0,0 +1,105 @@
+README
+======
+
+Here is a basic playbook (`deployment.yml`) to test the deployment of
+Avocado in a number of different environments, and installed via
+different methods.
+
+This code is intended to be used as regular playbook, so roughly the
+following steps should be followed:
+
+ 1) Install ansible. If you need help, refer to
+    https://docs.ansible.com/ansible/latest/installation_guide
+
+ 2) Adjust your inventory, that is, the hosts that will be used on the
+    playbook execution. The simplest way is to edit the `inventory`
+    file this directory.
+
+ 3) (OPTIONAL) adjust your variables on `vars.yml` or change the -e option via
+    command-line. Example::
+
+    $ ansible-playbook .... -e "method=pip"
+ 
+ 4) Run a playbook with::
+
+    $ ansible-playbook -i inventory -c local deployment.yml -e "method=pip"
+
+.. note:: Note that `-e` option it is not mandatory.
+
+Main variables
+==============
+
+Besides the basic variables (visit `vars.yml`) we have two main variables to
+control the execution model: `method` and `avocado_vt`.
+
+method
+------
+
+ 1. `pip`: Install Avocado and Plugins using pip on a Python virtual
+ environment (this is the default method).
+
+ 2. `copr`: Install Avocado and Plugins using RPM, from a Copr
```

Comment:
When the `copr` method is used on Fedora 31 (and later), this is what we get:

```
TASK [avocado : Install Avocado plugins available on Fedora only] ********************************************************************************
failed: [localhost] (item=python3-avocado-plugins-runner-remote) => {"ansible_loop_var": "item", "changed": false, "failures": [], "item": "python3-avocado-plugins-runner-remote", "msg": "Depsolve Error occured: \n Problem: conflicting requests\n  - nothing provides python3-fabric3 needed by python3-avocado-plugins-runner-remote-67.0-1.20190211gitcd4fdd51.fc30.noarch\n  - nothing provides python3-avocado = 67.0 needed by python3-avocado-plugins-runner-remote-67.0-1.20190211gitcd4fdd51.fc30.noarch\n  - nothing provides python3.7dist(fabric3) >= 1.1.4 needed by python3-avocado-plugins-runner-remote-67.0-1.20190211gitcd4fdd51.fc30.noarch\n  - nothing provides python3.7dist(fabric3) < 2.0.0 needed by python3-avocado-plugins-runner-remote-67.0-1.20190211gitcd4fdd51.fc30.noarch", "rc": 1, "results": []}
failed: [localhost] (item=python3-avocado-plugins-runner-vm) => {"ansible_loop_var": "item", "changed": false, "failures": [], "item": "python3-avocado-plugins-runner-vm", "msg": "Depsolve Error occured: \n Problem: conflicting requests\n  - nothing provides python3-avocado = 67.0 needed by python3-avocado-plugins-runner-vm-67.0-1.20190211gitcd4fdd51.fc30.noarch", "rc": 1, "results": []}
```

Which is expected because there's no `python3-avocado-plugins-runner-vm` on Fedora 31 and later.  This is similar to the previous issue about skipping package installs on some scenarios, but in this case, it seems necessary because the playbook will error out if this is not dealt with.

Reply:
Agree. Removed on v3.